common/src/main/java/com/gregtechceu/gtceu/core/MixinHelpers.java

```diff
@@ -70,7 +72,12 @@ public static void addMaterialBlockLootTables(Map<ResourceLocation, LootTable> l
         });
     }
 
-    public static List<PackResources> addDynamicData(Collection<PackResources> packs) {
+    @ExpectPlatform
+    public static void addFluidTexture(Material material, FluidStorage.FluidEntry value) {
+        throw new AssertionError();
+	}
+
+	public static List<PackResources> addDynamicData(Collection<PackResources> packs) {
```

Comment:
Not sure why, but this got merged with tab indents and displays kinda wonky on github.
```suggestion
    }

    public static List<PackResources> addDynamicData(Collection<PackResources> packs) {
```